presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -331,11 +340,13 @@ private PlanNodeStatsEstimate crossJoinStats(JoinNode node, PlanNodeStatsEstimat
     {
         private final PlanNodeStatsEstimate stats;
         private final Optional<Symbol> selectedClauseSymbol;
+        private final double selectedClauseSymbolNdv;
 
-        public AntiJoinStats(PlanNodeStatsEstimate stats, Optional<Symbol> selectedClauseSymbol)
+        public AntiJoinStats(PlanNodeStatsEstimate stats, Optional<Symbol> selectedClauseSymbol, double selectedClauseSymbolNdv)
```

Comment:
It is hard to follow. 
It looks like the commit mixes two things. 
1. Adding `DEFAULT_UNMATCHED_ANTI_JOIN_NDVS_COEFFICIENT`
2. refactor to add `selectdClauseSymbolNdv` to `AntiJoinStats`

I do not understand why we want `2`. It makes `AntiJoinStats` a junk bag. It does not seem to fit in here.